.ddev/config.yaml

```diff
@@ -3,7 +3,6 @@ type: drupal9
 docroot: docroot
 php_version: "7.4"
 webserver_type: apache-fpm
-dbimage: massgov/mysql-sanitized:super
```

Comment:
Do we know if this is an unintended compatibility break in ddev, or is there something "wrong" with how this was done before?

Reply:
We were relying on ddev internals. I worked with rfay on this new approach.